client/src/js/app.js

```diff
@@ -15,10 +15,26 @@ function bhimaconfig($routeProvider) {
     controller: 'editAccountBudget',
     templateUrl: 'partials/budget/edit/edit_budget.html'
   })
+  .when('/budgeting/analysis', {
```

Comment:
You added these controllers (analysis and create) twice.  Please remove the duplicates.